test/e2e/atlas/atlas_e2e_test_generator_test.go

```diff
@@ -282,7 +281,7 @@ func deleteOrgTeams(t *testing.T, cliPath string) {
 	var teams atlasv2.PaginatedTeam
 	require.NoError(t, json.Unmarshal(resp, &teams), string(resp))
 	for _, team := range teams.GetResults() {
-		assert.NoError(t, deleteTeam(team.GetId()))
+		require.NoError(t, deleteTeam(team.GetId()))
```

Comment:
can you rollback this change? currently this is intentionally to try and delete all instead of failing at the first error

Reply:
ah okay, I was getting errors with linter

Reply:
you have the old linter which had the false possitive, run `make setup` to update